gunicorn/config.py

```diff
@@ -203,6 +204,25 @@ def sendfile(self):
 
         return True
 
+    @property
+    def paste_global_conf(self):
+        raw_global_conf = self.settings['raw_paste_global_conf'].get()
+        if raw_global_conf is None:
+            return None
+
+        global_conf = {}
+        for e in raw_global_conf:
+            s = _compat.bytes_to_str(e)
+            try:
+                k, v = re.split(r'(?<!\\)=', s, 1)
```

Comment:
Ignore me. Right. The backslash here is not for character escape but for regular expression escape. Whole reason for the raw is to not have `\\\\`.

:space_invader: :moon: :rocket: